script.js

```diff
@@ -91,4 +91,26 @@ for (let i = 0; i < buttonsProject.length; i += 1) {
 popupExitButton.addEventListener('click', () => {
   popupWindow.style.display = 'none';
   backGround.style.display = 'none';
+});
+
+const contForm = document.getElementById('contact');
+const emailForm = document.getElementById('mail');
+const errorMessage = document.getElementById('error-message');
+
+const emailLowercase = (mail) => {
+  const itLowerc = new RegExp(/[A-Z]/g);
+  if (itLowerc.test(mail)) {
+    return false;
+  }
+  return true;
+};
+
+contForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (emailLowercase(emailForm.value.trim())) {
+    contForm.submit();
+    errorMessage.style.display = 'none';
+  } else {
+    errorMessage.style.display = 'block';
+  }
```

Comment:
- [ ] Your error message doesn't display when a wrong email is input and submit

Reply:
![image](https://user-images.githubusercontent.com/99298594/163285088-3fecd214-f32e-46f3-bee3-5ba845a52be6.png)
The image below shows the error message when you put your email wrong to the email input, please clarify your feedback to improve my code.

Reply:
Great job done, it works fine now